Dockerfile

```diff
@@ -39,6 +39,7 @@ ENV PROJ_LIB=/opt/conda/share/proj
 RUN conda install -c conda-forge matplotlib basemap cartopy python-igraph imagemagick pysal && \
     # b/142337634#comment22 pin required to avoid torchaudio downgrade.
     conda install -c pytorch pytorch torchvision "torchaudio>=0.4.0" cpuonly && \
+	conda install -c conda-forge --no-deps pixman==0.34 && \
```

Comment:
Use spaces here and align it with the previous command

Reply:
Could you add the following comment above:

```
# pixman 0.38.0 has a known issue causing issues with packages such as pycairo & openslide.
# See: https://gitlab.freedesktop.org/pixman/pixman/commit/8256c235d9b3854d039242356905eca854a890ba
```

Reply:
Added and changed as requested